src/test/java/com/checkmarx/cxintegrations/reposmanager/api/getrepos/getReposApiRunner.java

```diff
@@ -0,0 +1,12 @@
+package com.checkmarx.cxintegrations.reposmanager.api.getrepos;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        features = "src/test/resources/cucumber/get-repos-api.feature",
+        glue = {"com.checkmarx.cxintegrations.reposmanager.api.getrepos" } )
```

Comment:
```suggestion
        glue = {"com.checkmarx.cxintegrations.reposmanager.api.getrepos" } )
```
This line is not required.